packages/env/lib/build-docker-compose-config.js

```diff
@@ -183,7 +183,7 @@ module.exports = function buildDockerComposeConfig( config ) {
 		phpunitTag = '7' + phpunitPhpVersion;
 	} else if ( [ '7.2', '7.3', '7.4' ].indexOf( testsPhpVersion ) >= 0 ) {
 		phpunitTag = '8' + phpunitPhpVersion;
-	} else if ( testsPhpVersion === '8.0' ) {
+	} else if ( [ '8.0', '8.1', '8.2' ].indexOf( testsPhpVersion ) >= 0 ) {
 		phpunitTag = '9' + phpunitPhpVersion;
```

Comment:
This condition block seems wrong/out of date.
PHP 7.2 needs PHPUnit 8.x.
PHP 7.3 and higher should use PHPUnit 9.x

Also see: https://phpunit.de/supported-versions.html

Reply:
Yes, I saw that page.
But I was primarily focused on making wp-env work with PHP 8.2.
This slipped under the radar.
Thanks for pointing out.
Fixed in d7195bf64a5730fa4a5069cc02232e18edf6f713 and 26687328057b3c6052e65d7d1415ce2c20a9cde3

Reply:
🙌🏻